.github/workflows/test-src.yml

```diff
@@ -10,9 +10,6 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
```

Comment:
I believe have to change: `on:` trigger above. 
Otherwise package.json changes could break /src tests.